src/hooks/useIntersectionObserverRef.ts

```diff
@@ -17,7 +17,7 @@ const config: IntersectionObserverInit = {
  * @param {IntersectionObserverInit} options
  */
 function useIntersectionObserverRef(
-  callback: IntersectionObserverCallback,
+  callback?: IntersectionObserverCallback,
```

Comment:
Can we instead change this to `callback: IntersectionObserverCallback | undefined` ? This hook is practically useless without the callback but since we want to handle an `undefined` use case, let's put the types to match it?

Reply:
It looks good to be able to show more clearly what we want. I got it!